indexer/osm_editor.cpp

```diff
@@ -0,0 +1,109 @@
+#include "osm_editor.hpp"
+
+#include "indexer/feature_decl.hpp"
+#include "indexer/index.hpp"
+
+#include "std/map.hpp"
+#include "std/set.hpp"
+
+namespace osm
```

Comment:
If you think that osm::Editor is the correct name for the class, please, create a separate directory named "osm", and a separate library named "osm". And, of course, move editor.{cpp|hpp} to that directory.